pkg/syncer/storagepool/service.go

```diff
@@ -105,7 +105,11 @@ func InitStoragePoolService(ctx context.Context, configInfo *commontypes.ConfigI
 		defer diskDecommEnablementTicker.Stop()
 		clusterFlavor := cnstypes.CnsClusterFlavorWorkload
 		for ; true; <-diskDecommEnablementTicker.C {
-			coCommonInterface, _ := commonco.GetContainerOrchestratorInterface(ctx, common.Kubernetes, clusterFlavor, *coInitParams)
+			coCommonInterface, err := commonco.GetContainerOrchestratorInterface(ctx, common.Kubernetes, clusterFlavor, *coInitParams)
+			if err != nil {
+				log.Errorf("Failed to create CO agnostic interface. Error: %v", err)
+				break
```

Comment:
instead of `break` which would stop checking for feature enablement later, would it make more sense to `continue`, say for case where err is intermittent?